website/content/en/preview/upgrading/upgrade-guide.md

```diff
@@ -34,7 +34,7 @@ kubectl apply -f https://raw.githubusercontent.com/aws/karpenter{{< githubRelRef
 ```
 
 ### Upgrading to v0.34.0+
-* Karpenter now supports `nodepool.spec.disruption.budgets`. Since this requires an update to the Custom Resource, before upgrading, you should re-apply the new updates to the CRDs. Check out [Disruption Budgets]({{<ref "../concepts/disruption#disruption-budgets" >}}) for more.
+* Karpenter now supports `nodepool.spec.disruption.budgets`, which allows users to control the speed of disruption in the cluster. This will allow Karpenter to disrupt multiple batches of nodes simultaneously, which can result in overall quicker scale-down of your cluster. Karpenter will default to disrupting at most 10% of your nodes simultaneously per NodePool.   Since this requires an update to the Custom Resource, before upgrading, you should re-apply the new updates to the CRDs. Check out [Disruption Budgets]({{<ref "../concepts/disruption#disruption-budgets" >}}) for more. 
```

Comment:
One other thing: I'm not sure this makes it super clear IMO that when they apply this new behavior, this will be different from the existing behavior. To me, I'd make a callout that the existing behavior is that we disrupted a single node at a time and then tell them how they can keep the existing behavior using a budget that models that behavior

Reply:
There's no setting that would be perfectly equivalent, considering it was previously artificially bounded per-disruption method. I can call out which methods would be different. WDYT?

Reply:
I wonder if we just add detail here, then, that describes what the old behavior was here in a little more detail. Mainly want to callout here that when you upgrade to this newer version, you will see different behavior which will result in more aggressive disruption in general (unless we are talking about emptiness).